app/assets/stylesheets/mortgage_calculator/components/_stamp-duty-calc.scss

```diff
@@ -93,7 +93,9 @@ input.stamp-duty__input {
   width: 100%;
   position: relative;
   float: left;
+}
 
+.stamp-duty__input--wrapper--inline {
```

Comment:
Hi @JiggyPete Looking at the markup i believe this should be .stamp-duty__input-wrapper--inline:

```
Block: .stamp-duty
Element: __input-wrapper
Modifier: --inline
```